src/main/java/htsjdk/samtools/cram/compression/rans/E14.java

```diff
@@ -43,7 +43,7 @@ static int compress(final ByteBuffer in, final RansEncSymbol[][] syms,
         // Deal with the remainder
         l3 = 0xFF & in.get(in_size - 1);
         for (i3 = in_size - 2; i3 > 4 * isz4 - 2 && i3 >= 0; i3--) {
-            final int c3 = 0xFF & in.get(i3 > -1 ? i3 : 0);
+            final int c3 = 0xFF & in.get(i3);
```

Comment:
I thought this looked suspicious but I missed that the `for` loop directly restricts `i3`, e.g. `i3 >= 0`